wpilibc/src/main/native/cpp/simulation/ElevatorSim.cpp

```diff
@@ -75,9 +85,11 @@ units::ampere_t ElevatorSim::GetCurrentDraw() const {
   // is spinning 10x faster than the output.
 
   // v = r w, so w = v / r
+  double kA = 1.0 / m_plant.B(1, 0);
+  double kV = kA * m_plant.A(1, 1);
   units::meters_per_second_t velocity{m_x(1)};
   units::radians_per_second_t motorVelocity =
-      velocity / m_drumRadius * m_gearing * 1_rad;
+      units::radians_per_second_t{velocity.value() * kV * m_gearbox.Kv.value()};
```

Comment:
Fix the unit types instead of using `value()`. `kV` should have units of volts per meter per second, and `m_gearbox.Kv` has units of radians per second per volt.

Reply:
How do I declare kV (not the gearbox one) to have those units

Reply:
```cpp
using Kv_t = units::unit_t<units::compound_unit<units::volt, units::inverse<units::meters_per_second>>>;
Kv_t Kv;
```

Reply:
Thank you.